cg/meta/workflow/fluffy.py

```diff
@@ -124,17 +131,22 @@ def add_concentrations_to_samplesheet(
 
         sample_name_index = None
         sample_id_index = None
+        project_index = None
```

Comment:
How about utilising pydantic here so that the file gets validated at the same time? I think that Maya has a nice example for this in the lims package. Another example [here](https://github.com/Clinical-Genomics/cgmodels/blob/27792a4a747bc103a84bdc9f4cf2ce2e6d26214c/cgmodels/demultiplex/sample_sheet.py#L161)

Reply:
next time!